exec/csnex/Exec.cs

```diff
@@ -536,7 +539,19 @@ void JFCHAIN()
 
         void JUMPTBL()
         {
-            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+            ip++;
+            int val = Bytecode.Get_VInt(bytecode.code, ref ip);
+            Number n = stack.Pop().Number;
+            if (n.IsInteger() && !n.IsNegative()) {
+                int i = n;
+                if(i < val) {
```

Comment:
Inconsistent spacing after `if`

Reply:
Opps.  I think my Visual Studio formatter is still not quite perfect!  -- Fixed.